packages/pwa-kit-dev/CHANGELOG.md

```diff
@@ -1,3 +1,4 @@
+## v3.3.0 (Dec 08, 2023)
 ## v3.3.0-preview.1 (Dec 08, 2023)
 ## v3.3.0-preview.0 (Nov 30, 2023)
```

Comment:
For all changelog files, let's make sure that we remove any `-preview` nor `-dev` versions in them.